app/controllers/root_controller.rb

```diff
@@ -1,10 +1,15 @@
 class RootController < ApplicationController
   skip_before_action :authenticate_user!
-  skip_verify_authorized only: %i[index]
+  skip_verify_authorized only: %i[index up]
```

Comment:
Health check that is necessary to deploy! A simple `plain: ok` is good enough. Make sure to have no authorization requirements.